client/settings/loadable-account-section.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Same file as in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2024 - in there I also added some tests for it